Doc/faq/gui.rst

```diff
@@ -90,7 +90,7 @@ Kivy is free and open source software distributed under the MIT license.
 FLTK
 ----
 
-Python bindings for `the FLTK toolkit <http://www.fltk.org>`_, a simple yet
+Python bindings for `the FLTK toolkit <http://www.fltk.org/index.php>`_, a simple yet
```

Comment:
`/index.php` is an implementation detail. It just means that currently the site is implemented in PHP. After rewiting it on Python that link likely will become not valid.

Reply:
But that is an issue in that website, not in this docs, isn’t it?
If the goal of the PR is to avoid redirects, and the fltk website currently redirects `/` to `/whatever.implementiation.detail`, the change here is correct?